pages/docs/index.js

```diff
@@ -126,10 +126,11 @@ export default function DocsIndex() {
                   </div>
                   <div className="mt-12 grid gap-16 lg:grid-cols-3 lg:gap-x-5 lg:gap-y-12">
                     {section.pages.map((page) => (
+                      //added margin top to fix the issue
                       <Link
                         href={page.path}
                         key={page.name}
-                        className="border border-transparent hover:border hover:border-orange-600 transition-all duration-250 ease-linear rounded px-6 py-2 block"
+                        className="border border-transparent hover:border hover:border-orange-600 transition-all duration-250 ease-linear rounded px-6 py-2 mt-4 block"
```

Comment:
I don't think it need to be added to every link item? The ones on the 2nd row will now have a bigger gap at the top and that is not required. Suggest adding to the wrapping item